custom/opm/beneficiary.py

```diff
@@ -913,3 +914,175 @@ def __init__(self, case, report, child_index=1, **kwargs):
+        ('month', ugettext_lazy("Calendar month"), True, None)
+    ]
+
+    def __init__(self, case, report, child_index=1, awc_codes={}, **kwargs):
+        super(LongitudinalConditionsMet, self).__init__(case, report,
+                                                        child_index=child_index,
+                                                        awc_codes=awc_codes, **kwargs)
+        awc_data = awc_codes.get(self.awc_name, None)
+        self.gp = awc_data[1] if awc_data else EMPTY_FIELD
+        self.bank_branch_code = self.case_property('bank_branch_code', EMPTY_FIELD)
+        self.caste_tribe_status = self.get_value_from_form(PREG_REG_XMLNS, 'form/caste_tribe_status')
+        self.prev_pregnancies = self.get_value_from_form(PREG_REG_XMLNS, 'form/prev_pregnancies')
+        self.prev_live_births = self.get_value_from_form(PREG_REG_XMLNS, 'form/prev_live_births')
+        self.sons_alive = self.get_value_from_form(PREG_REG_XMLNS, 'form/sons_alive')
+        self.daughters_alive = self.get_value_from_form(PREG_REG_XMLNS, 'form/daughters_alive')
+        self.sum_children = self.get_value_from_form(PREG_REG_XMLNS, 'form/sum_children')
+        self.contact_phone_number = self.get_value_from_form(PREG_REG_XMLNS, 'form/contact_phone_number')
+        for idx in range(1, 4):
+            child_name = self.case_property('child%s_name' % str(idx), None)
+            child_age = self.child_age
+            if not child_name:
+                child_age = EMPTY_FIELD
+                child_name = EMPTY_FIELD
+            setattr(self, 'child%s_name' % str(idx), child_name)
+            setattr(self, 'child%s_age' % str(idx), child_age)
+            setattr(self, 'child%s_sex' % str(idx),
+                    self.case_property('child%s_sex' % str(idx), EMPTY_FIELD))
+        self.one = self.condition_image(C_ATTENDANCE_Y, C_ATTENDANCE_N, "पोषण दिवस में उपस्थित",
+                                        "पोषण दिवस में उपस्थित नही", self.preg_attended_vhnd)
+        self.two = self.condition_image(M_WEIGHT_Y, M_WEIGHT_N, "गर्भवती का वज़न हुआ",
+                                        "गर्भवती का वज़न नही हुआा", self.preg_weighed_trimestered(6))
+        self.two_two = self.condition_image(M_WEIGHT_Y, M_WEIGHT_N, "गर्भवती का वज़न हुआ",
+                                            "गर्भवती का वज़न नही हुआा", self.preg_weighed_trimestered(9))
+        self.three = self.three = self.condition_image(IFA_Y, IFA_N, "तीस आयरन की गोलियां लेना",
+                                                       "तीस आयरन की गोलियां नही लिया", self.preg_received_ifa)
+        self.four = self.condition_image(C_ATTENDANCE_Y, C_ATTENDANCE_N, "पोषण दिवस में उपस्थित",
+                                         "पोषण दिवस में उपस्थित नही", self.child_attended_vhnd)
+        self.five = self.condition_image(C_REGISTER_Y, C_REGISTER_N, "जन्म पंजीकृत",
+                                         "जन्म पंजीकृत नहीं", self.child_age == 6)
```

Comment:
Confirming that this can be done in transifex now.  No strong feelings about whether or not it's worth doing, given that the other reports are similarly hard-coded.